pkg/util/log/eventpb/contention_events.proto

```diff
@@ -0,0 +1,33 @@
+// Copyright 2021 The Cockroach Authors.
+//
+// Use of this software is governed by the CockroachDB Software License
+// included in the /LICENSE file.
+
+syntax = "proto3";
+package cockroach.util.log.eventpb;
+option go_package = "github.com/cockroachdb/cockroach/pkg/util/log/eventpb";
+
+import "gogoproto/gogo.proto";
+import "util/log/logpb/event.proto";
+
+// Category: Contention events
+// Channel: SQL_EXEC
+//
+// Aggregated information about contention events.
+
+// Notes to CockroachDB maintainers: refer to doc.go at the package
+// level for more details. Beware that JSON compatibility rules apply
+// here, not protobuf.
+// *Really look at doc.go before modifying this file.*
+
+
+// AggregatedContentionInfo is recorded periodically when contention events
+// are resolved.
+message AggregatedContentionInfo {
+  CommonEventDetails common = 1 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "", (gogoproto.embed) = true];
+  string waiting_stmt_fingerprint_id = 2  [(gogoproto.jsontag) = ",omitempty", (gogoproto.nullable) = false, (gogoproto.customtype) = "github.com/cockroachdb/redact.SafeString", (gogoproto.moretags) = "redact:\"nonsensitive\""];
```

Comment:
you don't need to set the type as `redactSafeString`. It will just work with `string` and you can save yourself the conversion.

Reply:
done.